cli/src/index.ts

```diff
@@ -35,9 +35,11 @@ program
       .default(false),
   )
   .addOption(new Option('-i, --ignore [paths...]', 'Paths to ignore').env('IMMICH_IGNORE_PATHS').default(false))
+  .addOption(new Option('--no-read-only', 'Import files without read-only protection, allowing Immich to manage them').default(true))
```

Comment:
I am confused by this option. It is called no read only but the default is true. What does that mean exactly?

Reply:
It looks like the commander library interprets `--no` prefixed options as the inverse. Settings default to true here is redundant as that is the default because it's the inverse. I can remove it to prevent confusion.

Reply:
Perfect, thanks